### PR TITLE
Imp export f1 distri

### LINKED ISSUE #327: Mejora de Traducciones
(boyscout required) Some parts of the page like https://github.com/gisce/powerp-docs/pull/326/files#diff-146a8cf2f5fb0bbf97c0d36bc285b254R34 are not well translated

### PR DESCRIPTION
**Pull request by guilleJB**
### Description:

Afegir la descripció de l'exportació 

### Tasks:

- [x] Docs
- [x] Traducciones
- [x] Fix traduciones ( Closes #327 )

### Links:

* [Distri facturacio #Exportacio Factures F1](http://builds.gisce.net/powerp-docs/powerp_imp_export_f1_distri/ca/distri/facturacion/#exportacio-factures-f1)
* [Distri facturacio ES #Exportacio Factures F1](http://builds.gisce.net/powerp-docs/powerp_imp_export_f1_distri/es/distri/facturacion/#exportacion-facturas-f1)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_export_f1_distri/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_export_f1_distri/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_export_f1_distri/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_export_f1_distri/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_export_f1_distri/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_export_f1_distri/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_export_f1_distri/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/facturacion.md [hunk 1]
```diff
@@ -387,6 +387,27 @@ Per exportar les factures generades en format XML F1 per enviar a les
...
+
+    En aquest camp indiquem per a quina comercialitzadora volem exportar els fitxers F1s.
+    Aquí podrem escollir o `Totes` les comercialitzadores o la comercialitzadora per la qual volem generar els fitxers
+
+2.  **Filtra per dates**
+
+    Serveix per filtrar entre quines dates de factura es vol exportar els fitxers F1s.
+    Per defecte el camp no està marcat exporta totes les facturas.
+    Si aquest camp està marcat només exportarpa les factures que compleixin el rang de dates indicades.
+
```
Comment (MiquelIR) [inline docs/distri/facturacion.md]: facturas -> factures
exportarpa -> exportarà
generar -> genera o generarà

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_export_f1_distri/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**